check-module-version/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Check module version
+description: >
+  Сhecks the equality of the version from Lua 
+  module's code and the last repository tag
+
+inputs:
+  module-name:
+    description: Lua module name as when using require() in tarantool
+    required: true
+  version-pre-excraction-hook:
```

Comment:
Is `''` the default value?

Reply:
Done, now an empty string is passed by default